assets/src/application/ui/input/InlineEditInput/types.ts

```diff
@@ -1,8 +1,11 @@
 import { InlineEditProps } from '@infraUI/inputs';
 
-export interface TextAreaProps extends Omit<InlineEditProps, 'inputType'> { tooltip?: string; }
+export interface TextAreaProps extends Omit<InlineEditProps, 'inputType'> {
+	tooltip?: string;
+}
 
 export interface TextProps extends Omit<InlineEditProps, 'inputType'> {
+	clampable?: boolean;
```

Comment:
🤔

Reply:
I also wasn't sure what is the best name, but check it before applying: https://en.wiktionary.org/wiki/clampable

Reply:
what does the 'clamp" property on the `Dotdotdot` component even do?
does that restrict the line length or the number of lines? (i'm guessing the latter)

if it simply cuts the line length, then maybe `truncate` would be a better name, but I suspect that it does more than that.

...

Just looked at the code for the `Dotdotdot` component and ya, the `clamp` property controls the number of lines displayed. 

What might be a better solution is instead of doing this:

```js
	if (clampable && value.length > 25) {
		return <Dotdotdot clamp={2}>{textInput}</Dotdotdot>;
	}
```

do something like:

```js
	if (lineCount && value.length > 25) {
		return <Dotdotdot clamp={lineCount}>{textInput}</Dotdotdot>;
	}
```

so instead of having a `clampable` boolean property we have a `lineCount` integer property that allows for greater control over how the text is displayed, which might be needed for other use cases.

@nicolad 

another thing that might be a good idea is to extract and consolidate the logic for the `Preview` component that is used in the `InlineEditText` and `InlineEditTextArea` components and then import back into those. Then this fix for number of lines displayed can cover both use cases.

Reply:
thx for the feedback, updated